src/main/scala/com/advancedtelematic/campaigner/daemon/DeviceInstallationReportListener.scala

```diff
@@ -40,4 +45,24 @@ class DeviceInstallationReportListener()(implicit db: Database, ec: ExecutionCon
         _log.info(s"Got DeviceUpdateReport for device ${msg.device.show} which is not scheduled by campaigner, ignoring this message.")
     }
   }
+
+  private def addDeviceToFailedGroup(msg: DeviceInstallationReport): Unit =
+    if (!msg.result.success) {
+      val failureCode = msg.result.code
+      msg.correlationId match {
+        case MultiTargetUpdateId(_) =>
```

Comment:
Shall we also handle this case in a manner it is done [here](https://github.com/advancedtelematic/campaigner/blob/8bdac1530aa081d5c0bd1963e644a715c453f420/src/main/scala/com/advancedtelematic/campaigner/http/CampaignResource.scala#L43)?

Reply:
I think we can safely ignore this branch, because we really just care about what happens with campaigns. AFAIK the `MultiTargetUpdateId` is used when you push an update for a single device from the DDV (device detailed view). And that doesn't mingle with our US.

Reply:
Okay, makes sense. Didn't know about the specifics of a multi-target update. Thank you!